### PR TITLE
change chart color

### PR DESCRIPTION
**Pull request by ryonryon**
@axross Can you check this?
image

### CODE REVIEW

**Comment by axross** [comment]
@ryotogashi Hmm these colors look really different by the application color scheme. Can we talk about it later? Even though the application looks simply red and monotone, this chart looks like brown or orange color.

**Comment by ryonryon** [comment]
@axross sure. I was thinking color and divide tag( more than 80%, 50%, 30%, less than 30%) will be changed. Dashboard chart color was one color so I did change.

**Comment by axross** [comment]
@ryotogashi I see.

**Comment by ryonryon** [comment]
@axross Can you check?
Color and "See card list" will be changed later.
image

**Comment by axross** [comment]
@ryotogashi Can you push today's update?

**Comment by ryonryon** [comment]
@axross sorry men, can you check these changes?

**Comment by ryonryon** [comment]
I shouldn't have changed this way. these changes are not only charts color.

File: lib/src/widgets/dashboard_screen/card_maturity_donut_chart.dart [hunk 1]
```diff
@@ -13,15 +13,16 @@ class CardMaturityDonutChart extends StatelessWidget {
...
 
 class MatureCount {
   final String title;
   final int maturity;
+  final charts.Color color;
 
-  MatureCount(this.title, this.maturity);
+  MatureCount(this.title, this.maturity, Color color)
+      : this.color = new charts.Color(
+            r: color.red, g: color.green, b: color.blue, a: color.alpha);
```
Comment (axross) [inline lib/src/widgets/dashboard_screen/card_maturity_donut_chart.dart]: Could you put a comma at the end of arguments and format (just saving the file) it?

File: lib/src/widgets/dashboard_screen/summary.dart [hunk 2]
```diff
@@ -23,132 +23,212 @@ class Summary extends StatelessWidget {
...
+                          mainAxisAlignment: MainAxisAlignment.center,
+                          children: <Widget>[
+                            Padding(
+                              padding: EdgeInsets.only(right: 24),
+                            ),
+                            Icon(
+                              Icons.fiber_manual_record,
+                              size: 12,
+                              color: SarakaColors.lightRed,
+                            ),
```
Comment (axross) [inline lib/src/widgets/dashboard_screen/summary.dart:123]: Use just `Container` and border radius instead

File: lib/src/widgets/dashboard_screen/summary.dart [hunk 3]
```diff
@@ -23,132 +23,212 @@ class Summary extends StatelessWidget {
...
-                    Row(
-                      mainAxisAlignment: MainAxisAlignment.center,
-                      children: <Widget>[
-                        Padding(
-                          padding: EdgeInsets.only(left: 60.0),
+                    FlatButton(
+                      child: Container(
+                        width: 200,
+                        decoration: BoxDecoration(
+                          borderRadius: BorderRadius.all(Radius.circular(20.0)),
```
Comment (axross) [inline lib/src/widgets/dashboard_screen/summary.dart]: Use `ShapeBorder` and `SuperellipseShape(borderRadius: BorderRadius.circular(24))` instead

File: lib/src/widgets/dashboard_screen/summary.dart [hunk 4]
```diff
@@ -23,132 +23,212 @@ class Summary extends StatelessWidget {
...
                             Text(
-                              'See card list',
+                              'SEE ALL ' +
+                                  totalCardsNumber.toString() +
+                                  ' CARDS',
                               overflow: TextOverflow.ellipsis,
-                              style: SarakaTextStyles.headlineSmall,
-                              textAlign: TextAlign.center,
+                              style: SarakaTextStyles.heading.copyWith(
+                                  fontSize: 20, color: SarakaColors.darkGray),
```
Comment (axross) [inline lib/src/widgets/dashboard_screen/summary.dart]: Please put a comma at the end

File: lib/src/widgets/dashboard_screen/summary.dart [hunk 5]
```diff
@@ -23,132 +23,212 @@ class Summary extends StatelessWidget {
...
                             ),
                             Icon(
-                              Icons.keyboard_arrow_down,
+                              Icons.arrow_forward_ios,
```
Comment (axross) [inline lib/src/widgets/dashboard_screen/summary.dart]: Is the feather icon's one looking not good?

File: lib/src/widgets/dashboard_screen/summary.dart [hunk 6]
```diff
@@ -23,132 +23,212 @@ class Summary extends StatelessWidget {
...
+                    ),
+                  ],
+                ),
               ],
             ),
           ),
+          Container(
+            padding: EdgeInsets.only(bottom: 80),
+            child: Text(
+              todayLearnNumber + ' cards you study for today',
```
Comment (axross) [inline lib/src/widgets/dashboard_screen/summary.dart]: Sounds kinda strange as English. I prefer `X cards to study today` or `X cards you will study today`

**Review by axross — APPROVED** [review]
Looks good to me apart from a few points I mentioned!

File: lib/src/widgets/dashboard_screen/summary.dart [hunk 7]
```diff
@@ -23,132 +23,212 @@ class Summary extends StatelessWidget {
...
-                                    .toList()
-                                    .length),
-                            new MatureCount(
-                                "Less than 30%",
+                Stack(
+                  alignment: Alignment.center,
+                  children: <Widget>[
+                    Container(
+                      height: 280.0,
+                      width: 280.0,
```
Comment (axross) [inline lib/src/widgets/dashboard_screen/summary.dart:43]: We need to change it to be responsible (use MediaQuery.size) in another pull request

**Comment by ryonryon** [comment]
@axross Can you check the last commit? I made another change.

File: lib/src/widgets/dashboard_screen/card_maturity_donut_chart.dart (hunk 1 above)
Comment (ryonryon) [reply lib/src/widgets/dashboard_screen/card_maturity_donut_chart.dart]: Done

File: lib/src/widgets/dashboard_screen/summary.dart (hunk 4 above)
Comment (ryonryon) [reply lib/src/widgets/dashboard_screen/summary.dart]: Done.

File: lib/src/widgets/dashboard_screen/summary.dart (hunk 2 above)
Comment (ryonryon) [reply lib/src/widgets/dashboard_screen/summary.dart:123]: @axross Instead of icon?

File: lib/src/widgets/dashboard_screen/summary.dart (hunk 5 above)
Comment (ryonryon) [reply lib/src/widgets/dashboard_screen/summary.dart]: mmm yeah, I Like feather one.

File: lib/src/widgets/dashboard_screen/summary.dart (hunk 6 above)
Comment (ryonryon) [reply lib/src/widgets/dashboard_screen/summary.dart]: Done.

File: lib/src/widgets/dashboard_screen/summary.dart (hunk 3 above)
Comment (ryonryon) [reply lib/src/widgets/dashboard_screen/summary.dart]: done.

File: lib/src/widgets/dashboard_screen/summary.dart [hunk 8]
```diff
@@ -10,145 +11,230 @@ class Summary extends StatelessWidget {
...
 
-  final String totalCardsNumber;
+  final int totalCardsNumber;
 
   final String todayLearnNumber;
 
   final List cardsMaturity;
 
+  double getMatureCardRatio() {
+    return cardsMaturity
```
Comment (axross) [inline lib/src/widgets/dashboard_screen/summary.dart]: Use `=>` instead

File: lib/src/widgets/dashboard_screen/summary.dart (hunk 2 above)
Comment (axross) [reply lib/src/widgets/dashboard_screen/summary.dart:123]: Yes

Comment (ryonryon) [reply lib/src/widgets/dashboard_screen/summary.dart:123]: Done.

**Pull request merged**